tests/Command/CreateSiteCommandTest.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\Test\PageBundle\Command;
+
+use PHPUnit\Framework\TestCase;
+use Prophecy\Argument;
+use Prophecy\Prophecy\ObjectProphecy;
+use Sonata\PageBundle\Command\CreateSiteCommand;
+use Sonata\PageBundle\Model\SiteManagerInterface;
+use Sonata\PageBundle\Tests\Model\Site;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Helper\HelperSet;
+use Symfony\Component\Console\Tester\CommandTester;
+
+/**
+ * @author Ahmet Akbana <ahmetakbana@gmail.com>
+ */
+class CreateSiteCommandTest extends TestCase
+{
+    /**
+     * @var Application|ObjectProphecy
+     */
+    private $application;
+
+    /**
+     * @var SiteManagerInterface|ObjectProphecy
+     */
+    private $siteManager;
+
+    protected function setUp()
+    {
+        $this->siteManager = $this->prophesize('Sonata\PageBundle\Model\SiteManagerInterface');
```

Comment:
please use `SiteManagerInterface::class` etc. where possible

Reply:
This was copied from existing command tests, but I changed it to use class 👍